pkg/yurthub/proxy/local/local.go

```diff
@@ -230,3 +231,15 @@ func copyHeader(dst, src http.Header) {
 		}
 	}
 }
+
+func WithFakeTokenInject(handler http.Handler, serializerManager *serializer.SerializerManager) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		ctx := req.Context()
+		info, _ := apirequest.RequestInfoFrom(ctx)
+		if info.Resource == "serviceaccounts" && info.Subresource == "token" {
+			klog.Infof("Find serviceaccounts token request, response will be wrapped.")
+			w = newFakeTokenWrapper(w, serializerManager, req)
```

Comment:
maybe we need to return `fake token` directly here instead of using `fakeTokenWrapper`, because we can not modify the header content-length in `fakeTokenWrapper.Write()`.

Reply:
OK. Now the fake token will be written directly to the response here